src/main/java/org/jitsi/videobridge/VideobridgeExpireThread.java

```diff
@@ -44,6 +47,23 @@
      */
     private final WeakReference<Videobridge> videobridge;
 
+    /**
+     * Property name for EXPIRE_CHECK_SLEEP_SEC for
+     * {@link #expireCheckSleepSec}.
+     */
+    public static final String EXPIRE_CHECK_SLEEP_SEC
+            = "org.jitsi.videobridge.EXPIRE_CHECK_SLEEP_SEC";
+
+    /**
+     * Default value for {@link #expireCheckSleepSec}.
+     */
+    private static final int EXPIRE_CHECK_SLEEP_SEC_DEFAULT = 60;
```

Comment:
Not really mandatory, but maybe we could replace the 60 with Channel.DEFAULT_EXPIRE.